make_queue/templates/make_queue/js/calendar.html

```diff
@@ -1,8 +1,8 @@
 {% include 'make_queue/js/reservation_rules.html' %}
 <script>
-    var canMakeReservation = {{ can_make_more_reservations|lower }};
-    var canIgnoreRules = {{ can_ignore_rules }};
-    var rules = [
+    let canMakeReservation = {{ can_make_more_reservations|lower }};
+    let canIgnoreRules = {{ can_ignore_rules }};
+    let rules = [
```

Comment:
This file is loaded async using ajax on the new reservation page. So all these `let` statements will result in errors each time the week is changed.